README.md

```diff
@@ -64,40 +65,115 @@ Flags:
 ```
 
 Every argument can be defined using environment variables using `ROSETTA_` prefix; and replacing `.` for `_`; for example:
-```
+
+```sh
 ROSETTA_DATADIR="/my/dir"
 ROSETTA_GETH_GENESIS="/path/to/genesis.json"
-``` 
+```
+
+## Running the Rosetta RPC Server
+
+Running the Rosetta RPC Server from scratch will take some time to sync, since it runs a full archive node in the background. While it may be possible to run the Construction API in the future with a non-archive node, this is still required by the Rosetta spec for the Data API implementation in order to perform balance reconciliation.
+
+### Version 1: Running from `rosetta` source code
+
+You will need the following three repositories cloned locally:
+
+* `rosetta` (this repo)
+* [`celo-monorepo`](https://github.com/celo-org/celo-monorepo)
+* [`celo-blockchain`](https://github.com/celo-org/celo-blockchain)
+
+You also need the following dependencies to be met:
+
+* `go >= 1.14`
+* `rust >= 1.41.0` (`blockchain` dependency)
+* `node = 10` (`celo-monorepo` dependency)
+* `golangci` ([installation instructions](https://golangci-lint.run/usage/install/#local-installation)) (linter dependency for the Makefile)
+
+#### Running on Alfajores (Testnet)
+
+Prerequisites:
+
+* Checkout `celo-monorepo` branch `alfajores` and run `yarn && yarn build --ignore docs`
+* Checkout `celo-blockchain` tag `v1.1.2` (`git fetch --all && git checkout v1.1.2`) (NOTE: check that this matches the version specified in the `rosetta` `go.mod` file) and `make all`
+* Set paths to `celo-monorepo` and `celo-blockchain` as `CELO_MONOREPO_PATH` and `CELO_BLOCKCHAIN_PATH` respectively (paths can be absolute or relative to the `rosetta` repo. If desired, add these lines to your bash profile:
+
+  ```sh
+  export CELO_MONOREPO_PATH=path/to/celo-monorepo
+  export CELO_BLOCKCHAIN_PATH=path/to/celo-blockchain
+  ```
+
+* Checkout `rosetta` tag `v0.7.6` (`git fetch --all && git checkout v0.7.6`) (or latest released tag) and `make gen-contracts && make all`
+* Run `make alfajores-env` to create an empty datadir with the genesis block (only needs to be run the first time, upon initializing the service)
+
```

Comment:
Suggest adding the following:

You should see output similar to this:

```sh
mkdir -p ./envs/alfajores
curl 'https://storage.googleapis.com/genesis_blocks/alfajores' > ./envs/alfajores/genesis.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:-100 12600  100 12600    0     0  36842      0 --:--:-- --:--:-- --:--:-- 36842
```

Reply:
✅ added for `rc1` as well